.github/scripts/backport/main.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
```

Comment:
Let's include set -e so that if a command exits with a non-zero status code, the script fails.